adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/impl/AmqpContext.java

```diff
@@ -105,7 +105,7 @@ final int getPayloadSize() {
      * @return The content type of the AMQP 1.0 message.
      */
     final String getMessageContentType() {
-        return message.getContentType();
+        return Optional.ofNullable(message.getContentType()).orElse(MessageHelper.CONTENT_TYPE_OCTET_STREAM);
```

Comment:
A hint in the javadoc about the default value would be good here (like in `MqttContext#contentType`).

Reply:
Sure. Will add.

Reply:
Done.